.editorconfig

```diff
@@ -0,0 +1,13 @@
+# top-most EditorConfig file
```

Comment:
nice, a new standard config file to define configuration common to all configs. this should probably go into dev-rel-docs: https://editorconfig.org/